### PR TITLE
New Java getting-started page

### LINKED ISSUE #1195: Create a Getting-started page for Java
Java task for #964.

@cartermp @svrnm: how about realizing the kind of Getting-started page you've envisioned in #964, but only for Java to start off with?

### PR DESCRIPTION
**Pull request by chalin**
- Closes #1195

Preview: https://deploy-preview-1224--opentelemetry.netlify.app/docs/instrumentation/java/getting-started/

/cc @svrnm

### CODE REVIEW

**Comment by svrnm** [comment]
Thanks for sharing @chalin!

A thought for #964: we could **always** use the grpc example app, since it exists for the other languages as well. Makes it also easier then to tell people to try out a combination of two languages.

**Review by mateuszrzeszutek — APPROVED** [review]

File: content/en/docs/instrumentation/java/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+Next, you'll use a [Java agent to automatically instrument](../automatic) the
+client and server at launch time. While you can [configure the Java agent][] in
+a number of ways, the steps below use environment variables.
+
+ 1. Download [opentelemetry-javaagent.jar][] from [Releases][] of the
+    `opentelemetry-java-instrumentation` repo. The JAR file contains the agent
+    and all automatic instrumentation packages. {{% alert color="info" %}}<i class="fas fa-edit"></i> Take note of the path to the JAR file.{{% /alert %}}
+ 2. Set and export variables that specify the Java agent JAR and a [console
+    trace exporter,][] using a notation suitable for your shell/terminal
```
Comment (trask) [inline content/en/docs/instrumentation/java/getting-started.md:33]: @chalin is this generally preferred to keep the comma inside of the link?

File: content/en/docs/instrumentation/java/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    environment &mdash; we illustrate a notation for bash-like shells:
+    ```console
+    $ export JAVA_OPTS="-javaagent:PATH/TO/opentelemetry-javaagent.jar"
```
Comment (trask) [inline content/en/docs/instrumentation/java/getting-started.md:36]: I like the uppercase `PATH/TO/`, going to steal this 😄

**Review by jack-berg — APPROVED** [review]

File: content/en/docs/instrumentation/java/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+rpc.grpc.status_code=0, rpc.service=helloworld.Greeter, thread.id=1,
+rpc.method=SayHello, rpc.system=grpc, thread.name=main, net.peer.port=50051},
+capacity=128, totalAddedValues=9}
+```
+
+## What next?
+
+For more:
+
+- Run this example with another [exporter][] for telemetry data.
```
[the remark this answers is not in the document]
Comment (chalin) [reply content/en/docs/instrumentation/java/getting-started.md:96]: Updated, thanks!

**Review by trask — APPROVED** [review]

**Review by austinlparker — APPROVED** [review]

**Comment by chalin** [comment]
Thanks y'all for the feedback. Merging.

**Pull request merged**